hardhat.config.ts

```diff
@@ -175,6 +184,14 @@ module.exports = {
           browserURL: 'https://sepolia-explorer.arbitrum.io/',
         },
       },
+      {
+        network: 'holesky',
+        chainId: 17000,
+        urls: {
+          apiURL: "https://api-holesky.etherscan.io/api",
```

Comment:
@RohitAudit 
is this our holesky server endpoint for apiURL?

Reply:
this is for verification of contracts on etherscan